### DISCUSSION TITLE
월드 등록 업로드 넘어가지지않고 무한로딩중입니다.

### DISCUSSION THREAD

**Discussion by heavybowgun** [naverz/zepeto-studio-kor#909] [5. Q&A]
#901 글과 똑같은 문제를 겪는중입니다.

댓글에 해결방안을 올려주셔서 그대로 해봤지만 여전히 무한로딩중입니다.

현재 다른 해결 방안이 있을까요? 

사용한 스크립트는 멀티서버, 카메라기능 만 넣은 상태이고
제페토패키지파일 용량은 74mb 입니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 저희쪽에서는 재현이 되지 않고 있어서 패키지 파일을 아래 이메일로 전달 주실 수 있을까요?
[dl_zepeto_evangelist@naverz-corp.com](mailto:dl_zepeto_evangelist@naverz-corp.com) 입니다.
감사합니다.

**Reply by heavybowgun** [reply]
메일 보냈습니다.

**Reply by Zepeto-TechEvangelist** [reply]
보내주신 파일 기반으로 내부에서 확인 중에 있습니다.
원인 파악 되는대로 연락 드리겠습니다.
감사합니다.

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 무한 로딩 관련 내부 로직 수정하였습니다.
전달 주신 패키지 파일을 다시 업로드 하시면 정상적으로 등록될 것으로 보여집니다.
감사합니다.

**Reply by heavybowgun** [reply]
패키지 파일 다시 올려봤지만 똑같은 현상이 계속 나오네요.

아예 유니티에서 퍼블리싱 다시하고 올려야 할까요?

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 패키지 파일 등록시 패키지 정보를 시스템 쪽에 회신해줄때 오류가 있는 부분을 확인하여 수정하였습니다.
불편 드려 죄송합니다.
전달 주신 패키지 파일을 다시 업로드 하시면 정상적으로 등록될 것으로 보여집니다.
감사합니다.
